lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Changing the `include?` types to
```TypeScript
include?: Includeable[] | { all: true }[];
```
nets:
```
test/include.ts(27,5): error TS2345: Argument of type '{ include: { all: true; include: { model: typeof AssociatedModel; where: { state: Col; }; limit: ...' is not assignable to parameter of type 'FindOptions'.
  Types of property 'include' are incompatible.
    Type '{ all: true; include: { model: typeof AssociatedModel; where: { state: Col; }; limit: number; sep...' is not assignable to type 'Includeable[] | { all: true; }[]'.
      Type '{ all: true; include: { model: typeof AssociatedModel; where: { state: Col; }; limit: number; sep...' is not assignable to type '{ all: true; }[]'.
        Type '{ all: true; include: { model: typeof AssociatedModel; where: { state: Col; }; limit: number; sep...' is not assignable to type '{ all: true; }'.
          Object literal may only specify known properties, and 'include' does not exist in type '{ all: true; }'.
```
thus preventing from the `{ all: true, include: ...}` example. Since you've stated that this should never be the case, I tried to reproduce it and have `tsc` throw an error when providing additional properties.